apps/api/src/app/environments-v1/usecases/output-renderers/in-app-output-renderer.usecase.ts

```diff
@@ -13,21 +13,21 @@ export class InAppOutputRendererUsecase {
       subject: inApp.subject,
       body: inApp.body,
       avatar: inApp.avatar,
-      primaryAction: inApp.primaryAction
+      primaryAction: inApp.buttonSettings?.primaryAction
```

Comment:
In `@novu/framework`, we already have an inApp output schema. We should be consistent and reuse it. PTAL [here](https://github.com/novuhq/novu/blob/4522dd303246b176c95d378549f3c383e1a6f8d2/packages/framework/src/schemas/steps/channels/in-app.schema.ts#L54).

Reply:
we are using that output schema, you are commenting on the renderrer which is using the control schema, 
control schema <> output schema
There is no way to implement the UI schema we talked about where a component == 1 control in that way,
specifically the button dropdown, it's a single UI item responsible for 8 diffrent elements and 2 level one elemnts, it must be nested to have a consistent contract

Reply:
For posterity, this was resolved offline.